packages/web-components/src/components/global-banner/global-banner.ts

```diff
@@ -130,7 +130,9 @@ class C4DGlobalBanner extends StableSelectorMixin(LitElement) {
    */
   _renderAsStatic() {
     return html`
-      <div class="${prefix}--global-banner-layout-container">
+      <div
+        class="${prefix}--global-banner-layout-container"
+        part="banner-container">
```

Comment:
Looking at the method this is part of, I can tell we either render this whole component as a static element, or as a link. Since this is the outermost wrapper, I think "container" is a sufficient name. We can also add a more specific name in case someone wants to target this only when it is rendered as a static element.
```suggestion
        part="container container--static">
```